nextflow.config

```
@@ -56,6 +56,10 @@ params {
     filter_ssu        = null
     min_len_asv       = null
     max_len_asv       = null
+    filter_codons     = null
+    orf_start         = null
```

Adding default is more explicit and helpful, imho
```suggestion
    orf_start         = 1
```